deploy/charts/openebs-monitoring/dashboards/Control plane/openebs-main-dashboard.json

```diff
@@ -0,0 +1,2187 @@
+            "hide": false,
+            "interval": "",
+            "legendFormat": "",
+            "refId": "C",
+            "valueHandler": "Number Threshold"
+          },
+          {
+            "expr": "",
+            "interval": "",
+            "legendFormat": "",
+            "refId": "A"
+          }
+        ],
+        "thresholds": "0.9,0.99",
+        "timeFrom": "1m",
+        "timeShift": null,
+        "title": "OpenEBS data plane availability over [$duration]",
+        "type": "singlestat",
+        "valueFontSize": "80%",
+        "valueMaps": [],
+        "valueName": "current"
+      },
+      {
+        "cacheTimeout": null,
+        "datasource": "Prometheus",
+        "description": "This panel shows if there is any unhealthy OpenEBS  replicas on $cluster cluster.\n\n",
+        "fieldConfig": {
+          "defaults": {
+            "custom": {},
+            "mappings": [
+              {
+                "from": "",
+                "id": 0,
+                "operator": "",
+                "text": "Healthy",
+                "to": "",
+                "type": 1,
+                "value": "3"
+              },
+              {
+                "from": "",
+                "id": 1,
+                "operator": "",
+                "text": "Unhealthy",
+                "to": "",
+                "type": 1,
+                "value": "2"
+              }
+            ],
+            "nullValueMode": "connected",
+            "thresholds": {
+              "mode": "absolute",
+              "steps": [
+                {
+                  "color": "#d44a3a",
+                  "value": null
+                },
+                {
+                  "color": "semi-dark-green",
+                  "value": 3
+                }
+              ]
+            },
+            "unit": "none"
+          },
+          "overrides": []
+        },
+        "gridPos": {
+          "h": 9,
+          "w": 12,
+          "x": 12,
+          "y": 34
+        },
+        "id": 10,
+        "interval": null,
+        "links": [],
+        "maxDataPoints": 100,
+        "options": {
+          "colorMode": "background",
+          "fieldOptions": {
+            "calcs": [
+              "lastNotNull"
+            ]
+          },
+          "graphMode": "none",
+          "justifyMode": "auto",
+          "orientation": "horizontal",
+          "reduceOptions": {
+            "calcs": [
+              "mean"
+            ],
+            "fields": "",
+            "values": false
+          },
+          "textMode": "auto"
+        },
+        "pluginVersion": "7.1.1",
+        "targets": [
+          {
+            "expr": "(openebs_total_replica_count{cluster=\"$cluster\"})",
+            "instant": true,
+            "interval": "",
+            "legendFormat": "{{pod}}",
+            "refId": "B"
+          }
+        ],
+        "timeFrom": null,
+        "timeShift": null,
+        "title": "$cluster OpenEBS replicas",
```

Comment:
This is not required in OpenEBS control plane dashboard. We can have specific dashboards for them.